### PR TITLE
Bundle

### PR DESCRIPTION
**Pull request by nofun97**
Part of #584 

Changes proposed in this pull request:
- adds `arrai bundle` command that bundles all dependencies into a runnable format

Checklist:
- [x] Added related tests
- [ ] Made corresponding changes to the documentation

### CODE REVIEW

File: cmd/arrai/bundle.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"github.com/arr-ai/arrai/tools"
+	"github.com/urfave/cli/v2"
+)
+
+const bundledType = ".arraiz"
+
+var bundleCommand = &cli.Command{
+	Name:    "bundle",
+	Aliases: []string{"b"},
+	Usage:   "bundle arrai script and its dependencies into a runnable file. It outputs .arraiz file.",
```
Comment (orlade-anz) [inline cmd/arrai/bundle.go:21]: Nit: 
```suggestion
	Usage:   "bundles an arr.ai script and its imports into an .arraiz file that can be `arrai run`.",
```

**Review by orlade-anz — COMMENTED** [review]
Nice 👍

**Review by marcelocantos — APPROVED** [review]
Great work!

**Review by orlade-anz — CHANGES_REQUESTED** [review]

**Comment by orlade-anz** [comment]
I don't understand most of this but it looks good!

**Comment by nofun97** [comment]
> I don't understand most of this but it looks good!
I should probably add a documentation then. Will add a design doc.

**Comment by orlade-anz** [comment]
> I should probably add a documentation then. Will add a design doc.

I applaud your enthusiasm, and it would be nice, but don't invest too much in it since this is a somewhat temporary solution.

**Review by orlade-anz — APPROVED** [review]

**Pull request merged**